app/api/events.py

```diff
@@ -106,8 +160,25 @@ def before_get_object(self, view_kwargs):
                 else:
                     view_kwargs['id'] = None
 
-    decorators = (jwt_required, )
+    def before_update_object(self, obj, data, view_kwargs):
+        if data['starts_at']:
+            data['starts_at'] = data['starts_at'].replace(tzinfo=None)
+        if data['ends_at']:
+            data['ends_at'] = data['ends_at'].replace(tzinfo=None)
+
```

Comment:
@LuD1161 I don't think this fixes the timezone aware issue. I checked, it still gives UTC time +00.00 even when the timezone is different